src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx

```diff
@@ -1061,9 +1071,9 @@ Conditions, violations, incidents, and issues all work for any entity.
     id="cust-evnts"
     title="Custom events"
   >
-    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic 🟡
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: 👀 
 
-    You can report custom events from OpenTelemetry by using the New Relic [Event API](/docs/data-apis/ingest-apis/introduction-event-api/). Alternatively, you can report on custom events using the OpenTelemetry events API via OTLP. These events are currently stored and queried as New Relic logs. For more about logs, see [New Relic data types](/docs/data-apis/understand-data/new-relic-data-types/#log-data).
+    You can report custom events from OpenTelemetry by using the New Relic [Event API](/docs/data-apis/ingest-apis/introduction-event-api/). Alternatively, you can report on custom events using the experimental OpenTelemetry events API via OTLP. These events are currently stored and queried as New Relic logs. For more about logs, see [New Relic data types](/docs/data-apis/understand-data/new-relic-data-types/#log-data).
```

Comment:
@rhetoric101 This needs tweaking with Mark's help before we publish

Reply:
@rkbodenner I chopped Mark's information in half, and put the second half here.